wee_slack.py

```diff
@@ -2964,6 +2965,52 @@ def me_command_cb(data, current_buffer, args):
     return w.WEECHAT_RC_OK_EAT
 
 
+def command_register(data, current_buffer, args):
+    e = EVENTROUTER
+    team = e.weechat_controller.buffers[current_buffer].team
```

Comment:
My suggestion of printing to the team wasn't that good after all. If you aren't connected to a team already, this will fail.

This command doesn't really have any relevance to the possibly existing teams or where you run it though, so I think we should just always print to the core buffer for this command. That can be done with `w.prnt("", message)`.

Reply:
Done.